lib/tasks/create_printers.rake

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Can only run in production, unless config for auto_create_printer_in_cupsd is changed
+namespace :printers do
+  desc 'generate list of printers'
+  task create: :environment do |_t|
+    existing_printers = `lpstat -a`.split("\n")
```

Comment:
The below is doing the same as in app/models/printer.rb
Maybe that method can be called from here?